README.md

```diff
@@ -8,8 +8,7 @@ your git repositories.
 ## Install
 
 ```
-$ curl https://raw.githubusercontent.com/joshfriend/hooker/master/hooker.sh
-$ ./hooker.sh --install
+$ bash <(curl -s https://raw.githubusercontent.com/joshfriend/hooker/master/hooker.sh) --install
```

Comment:
And then it might make sense to split up `hooker.sh` into `install.sh` and `hooker.sh`.